idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -42,13 +42,16 @@ export default PageView.extend({
       hook: 'feedback'
     }],
 
-    'model.modified_date': {
-      type: (el, value) => new Date(value),
+    'model.modified': {
+      type: (el, value) => {
+        const d = new Date(value);
+        el.innerHTML = `${d.getMonth() + 1} / ${d.getDate()} / ${d.getFullYear()}`;
```

Comment:
toLocaleDateString is not supported in safari :( I can look around for a little shim for it